### DISCUSSION TITLE
Component: without webpack and react

### DISCUSSION THREAD

**Discussion by jehon** [cypress-io/cypress#16923] [Component Testing] [answered]
Hello

I wonder if I can use Cypress "component" testing **without** using WebPack and **without** using React.

I am developping native Web Components (extends HTMLElement), and thus I don't need transpilation (webpack) or anything...

Reading the doc and looking at the plugin didn't help.

Anyone have such setup?

Thanks

**Comment by lmiller1990** [comment, marked as answer]
You need a dev server to serve the component in Component Testing now. You can definitely do Web Components + a basic webpack config.

Here's a simple example for web components: https://github.com/lmiller1990/cypress-web-components-example
